instance/models/mixins/openedx_database.py

```diff
@@ -377,6 +377,14 @@ def _get_rabbitmq_settings(self):
             "XQUEUE_RABBITMQ_HOSTNAME": self.rabbitmq_server.instance_host,
             "XQUEUE_RABBITMQ_PORT": self.rabbitmq_server.instance_port,
             "XQUEUE_RABBITMQ_TLS": True,
+            "XQUEUE_SESSION_ENGINE": "django.contrib.sessions.backends.cache",
+            "XQUEUE_CACHES": {
+                "default": {
+                    "BACKEND": "django.core.cache.backends.memcached.MemcachedCache",
+                    "KEY_PREFIX": "xqueue",
+                    "LOCATION": "localhost:11211",
```

Comment:
Can we maybe default to `"{{ EDXAPP_MEMCACHE }}"` here, to explicitly use the same memcached as edxapp?  Not sure which version is less confusing, but I feel it could be a bit error prone to hardcode the location of memcached here.

Reply:
Yep, that's fine with me. Note that upstream uses different vars for each role requiring memcache, i.e. `INSIGHTS_MEMCACHE`, `DISCOVERY_MEMCACHE`, and `CREDENTIALS_MEMCACHE`, so hopefully this is not confusing to have here.